content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -495,8 +495,11 @@ based on setting `securityContext` within the Pod's `.spec`.
 
 ## Annotations used for audit
 
+<!-- sorted by annotation -->
```

Comment:
Aside, the collation order I have in mind is the domain name part, with labels reversed, followed by the annotation key.